Documentation/openshiftdocs/modules/userworkloadconfiguration.adoc

```diff
@@ -1,24 +1,27 @@
-:_content-type: ASSEMBLY
-include::_attributes/common-attributes.adoc[]
-:context: configmap-reference-for-cluster-monitoring-operator
+// DO NOT EDIT THE CONTENT IN THIS FILE. It is automatically generated from the 
+	// source code for the Cluster Monitoring Operator. Any changes made to this 
+	// file will be overwritten when the content is re-generated. If you wish to 
+	// make edits, read the docgen utility instructions in the source code for the 
+	// CMO.
+	:_content-type: ASSEMBLY
 
 == UserWorkloadConfiguration
 
 === Description
 
-UserWorkloadConfiguration defines configuration that allows users to customise the monitoring stack responsible for user-defined projects through the user-workload-monitoring-config ConfigMap in the openshift-user-workload-monitoring namespace
+The `UserWorkloadConfiguration` resource defines the settings responsible for user-defined projects in the `user-workload-monitoring-config` config map  in the `openshift-user-workload-monitoring` namespace. You can only enable `UserWorkloadConfiguration` if you first set `enableUserWorkload` to `true` in the `openshift-monitoring` config map.
```

Comment:
You can only enable `UserWorkloadConfiguration` if you first set `enableUserWorkload` to `true` in the `openshift-monitoring` config map.
->
You should set `enableUserWorkload` to `true` in the `openshift-monitoring` config map first, then you can enable `UserWorkloadConfiguration`.

Reply:
Revised per your text